src/aiortc/contrib/media.py

```diff
@@ -18,11 +18,21 @@
 from ..mediastreams import AUDIO_PTIME, MediaStreamError, MediaStreamTrack, KeypointsFrame
 
 from first_order_model.fom_wrapper import FirstOrderModel
-config_path = '/data/vibhaa/aiortc/nets_implementation/first_order_model/config/api_sample.yaml'
+
+# instantiate and warm up the model
+"""
+time_before_instantiation = time.time()
+zero_array = np.zeros((256, 256, 3))
+"""
+config_path = '/home/ubuntu/aiortc/nets_implementation/first_order_model/config/api_sample.yaml'
 model = FirstOrderModel(config_path)
+zero_kps = model.extract_keypoints(zero_array)
+model.update_source(zero_array, zero_kps)
+model.predict(zero_kps)
```

Comment:
maybe we need more warm-up

Reply:
current version of code has this - will be fixed in subsequent commits